datalake/src/main/java/com/sequenceiq/datalake/controller/sdx/SdxController.java

```diff
@@ -184,6 +186,13 @@ public ObjectStorageValidateResponse validateCloudStorage(@ValidStackNameFormat
                 sdxValidateCloudStorageRequest.getDataAccessRole(), sdxValidateCloudStorageRequest.getRangerAuditRole());
     }
 
+    @Override
+    @CheckPermissionByAccount(action = AuthorizationResourceAction.DESCRIBE_DATALAKE)
```

Comment:
there is no account level role with this permission, authz with `@CheckPermissionByAccount` won't work in this case, you should use field clusterName of requestObject to check permission => use annotation `@CheckPermissionByRequestProperty(path = "clusterName", type = NAME, action = DESCRIBE_DATALAKE)`

Reply:
addressed.